internal/pkg/commands/cmd_runner_helper_unix.go

```diff
@@ -1,4 +1,4 @@
-package utils
+package commands
```

Comment:
No build tags here for !windows?

Reply:
The name of the file works like the tags. So I removed the unnecessary tags